test/mpi/util/mtest.c

```diff
@@ -1093,7 +1093,7 @@ void MTestPrintfMsg(int level, const char format[], ...)
     va_list list;
     int n;
 
-    if (verbose && level >= verbose) {
```

Comment:
I think the idea here is if the message is above the level of verbosity set, then it won't be printed. The existing test looks correct to me.